### DISCUSSION TITLE
How to use sections in the admonition blocks?

### DISCUSSION THREAD

**Discussion by hellt** [squidfunk/mkdocs-material#7473] [Q&A] [answered]
Hi all,
I am a big fan of collapsible admonition blocks. It helps with providing additional context that doesn't need to be expanded by default and doesn't obstruct the page view unless a reader decides they want to dive in.

The challenge with admonition blocks that I have is that I don't know how to make use of headings/sections in it, that won't be visible in the main nav.

Consider the following example:

<img width="922" alt="image" src="https://github.com/user-attachments/assets/bfaf26da-e14e-4212-bec0-9534b5964bca">

Here I need to provide the sections (Limitations and Known Issues) in the admonition block, but I do not want them to be visible in the right handside ToC.

Any chance this can be achieved?

**Comment by squidfunk** [comment, marked as answer]
This was already asked before a long, long time ago 😉 Closing as duplicate:

- https://github.com/squidfunk/mkdocs-material/discussions/4015
- https://github.com/squidfunk/mkdocs-material/issues/313#issuecomment-299698639

**Reply by hellt** [reply]
oh sweet, html tags it is then. Thanks!
